.travis.yml

```diff
@@ -14,16 +14,14 @@ branches:
 
 matrix:
   include:
-    - php: 5.3
-      env: DEPENDENCIES='low'
     - php: 5.6
       env: SYMFONY_VERSION='2.3.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.7.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.8.*'
     - php: 7.0
-      env: SYMFONY_VERSION='3.0.*'
+      env: SYMFONY_VERSION='^3.0'
```

Comment:
I would remove this job entirely instead. Testing against precise minor versions of Symfony makes sense only for LTS versions. Running against the latest Symfony version is already done by normal jobs